src/config/dynamic-theme-fixes.config

```diff
@@ -20671,19 +20690,32 @@ a[data-testid] > svg {
     fill: ${goldenrod} !important;
 }
 #xwd-board rect[class^="xwd__cell--cell"] {
-    fill: ${darkgrey} !important;
+    fill: #595963 !important;
```

Comment:
If you want to have this color in dark scheme then use `${#ada59b}` - this will do lighter cell in light scheme.